packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswerContent/templates/generatedMarkdownContent.css

```diff
@@ -100,6 +100,7 @@ blockquote {
 }
 
 .scrollable-table {
+  margin-top: var(--lwc-spacingLarge, 1.5rem);
```

Comment:
the removal of pre-wrap on white-space was causing a lack of margin on top of tables, so I fix it here. We were already setting margin on the bottom but not on the top.

before:

<img width="635" alt="image" src="https://github.com/user-attachments/assets/d9f04a0b-e071-46f9-a14d-f976de484b1f">


now:

<img width="1072" alt="image" src="https://github.com/user-attachments/assets/7cfd301f-c5f3-4a22-99fd-44ba81e725de">